distutils is deprecated in python 3.10, which means it imports but prints warnings

--- a/libraries/libapparmor/m4/ac_python_devel.m4
+++ b/libraries/libapparmor/m4/ac_python_devel.m4
@@ -69,7 +69,7 @@
         # Check if you have distutils, else fail
         #
         AC_MSG_CHECKING([for the distutils Python package])
-        ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
+        ac_distutils_result=`$PYTHON -W ignore -c "import distutils" 2>&1`
         if test -z "$ac_distutils_result"; then
                 AC_MSG_RESULT([yes])
         else
